plugins/node/opentelemetry-instrumentation-redis-4/README.md

```diff
@@ -17,8 +17,7 @@ npm install --save @opentelemetry/instrumentation-redis-4
 
 ### Supported Versions
 
-This package supports `redis@^4.0.0`
-For versions `redis@^2.6.0` and `redis@^3.0.0`, please use `@opentelemetry/instrumentation-redis`
```

Comment:
This extra info about redis v2 and v3 was useful, no?

Reply:
I was using this info to create automatic READMEs for the "@opentelemetry/auto-instrumentation-node" package, and it causes the text to be confusing, as this information is not relevant for a list of instrumentations listing both 
`redis` and `redis-4` one next to each others. the information about supported versions is covered in #2196.

My future plan is to merge `redis-4` into `redis`. I was the one to introduce it back then, as it felt right at the moment, but I think it harms usability and I want to revert it and use one package for all redis versions.

Reply:
Reverted this change. Will address the text in a different PR for merging both redis instrumentation packages